apps/meteor/client/views/root/MainLayout/LayoutWithSidebar.tsx

```diff
@@ -1,12 +1,20 @@
 import { Box } from '@rocket.chat/fuselage';
 import { useLayout, useSetting, useCurrentModal, useRoute, useCurrentRoutePath } from '@rocket.chat/ui-contexts';
 import type { ReactElement, ReactNode } from 'react';
-import { useEffect, useRef } from 'react';
+import { useEffect, useRef, version } from 'react';
 
 import AccessibilityShortcut from './AccessibilityShortcut';
 import { MainLayoutStyleTags } from './MainLayoutStyleTags';
 import Sidebar from '../../../sidebar';
 
+const inertBooleanSupported = Number(version.split('.')[0]) >= 19;
+
+/**
+ * Before version 19, react JSX treats empty string "" as truthy for inert prop.
+ * @see {@link https://stackoverflow.com/questions/72720469}
+ * */
+const isInert = inertBooleanSupported ? (x: boolean) => x : (x: boolean) => (x ? '' : undefined);
+
```

Comment:
I believe this will need to be replicated to [LayoutSidebarV2](https://github.com/RocketChat/Rocket.Chat/blob/9af2c11d64b85abd74989777eee93f48c0684ec0/apps/meteor/client/views/root/MainLayout/LayoutWithSidebarV2.tsx). It might be a good idea to have this as an util and use it in both files.